vault/resource_raft_snapshot_agent_config.go

```diff
@@ -429,7 +432,8 @@ func readSnapshotAgentConfigResource(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
-	if val, ok := resp.Data["google_endpoint"]; ok {
+	// Vault is returning 'false' for this instead of null.
```

Comment:
This is spooky and should probably be looked into on the Vault side. When this string isn't set, Vault returns `false` instead of `null` or an empty string like the other raft snapshot storage providers. Handling it here in the provider for now though.

Reply:
Can you confirm where you're seeing this returned as `false`? I modified the if case to log the Google Endpoint in case it was non-null and didn't see any logs. The API response to the TF Provider also doesn't seem to have a `google_endpoint` field in the response from Vault: 

<img width="1003" alt="Screen Shot 2021-09-28 at 3 04 46 PM" src="https://user-images.githubusercontent.com/86625824/135172471-472c4b71-8d93-4341-9b61-656fe818e0fb.png">

Reply:
Does this happen only when the storage type is `google_gcs` and the endpoint isn't specified?